### PR TITLE
chore - remove setContracts function

### PR DESCRIPTION
**Pull request by ilinzweilin**
- the function is not needed anymore, since we won't derive addresses based on root anymore

### CODE REVIEW

File: src/test/utils.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export class TestProvider {
   }
 }
 
-export async function createTinlake(usr: Account, testConfig: any) : Partial<ITinlake> {
+export function createTinlake(usr: Account, testConfig: any) : Partial<ITinlake> {
```
Comment (philipstanislaus) [inline src/test/utils.ts:38]: Can we add type info to the config Paramus and require the contract addresses to be present?

File: src/test/addresses.json [hunk 2]
```diff
@@ -1,36 +1,28 @@
...
+  "COLLATERAL_NFT":"",
+  "COLLATERAL_NFT_DATA":"",
+  "ROOT_CONTRACT":"",
+  "PROXY":"",
+  "PROXY_REGISTRY":"",
+  "ACTIONS":"",
+  "BORROWER_DEPLOYER":"",
+  "LENDER_DEPLOYER":"",
+  "NFT_FEED":"",
+  "GOVERNANCE":""
```
Comment (philipstanislaus) [inline src/test/addresses.json:27]: Brauchen wir diese Datei noch?

File: src/test/config.ts [hunk 3]
```diff
@@ -23,4 +20,18 @@ const testConfig = {
...
+  rpcUrl: string;
+  godAccount: Account;
+  gasLimit: number;
+  gasPrice: number;
+  transactionTimeout: number;
+  contractAddresses: ContractAddresses;
+  contractAbis: ContractAbis;
+  SUCCESS_STATUS: '0x1';
+  FAIL_STATUS: '0x0';
+  FAUCET_AMOUNT: string;
```
Comment (philipstanislaus) [inline src/test/config.ts:33]: Are they really all required?

File: src/test/addresses.json (hunk 2 above)
Comment (ilinzweilin) [reply src/test/addresses.json:27]: hab sie als template fuer das test setup drin gelassen :)

File: src/test/config.ts (hunk 3 above)
Comment (ilinzweilin) [reply src/test/config.ts:33]: for the tests, yes

**Pull request merged**